### PR TITLE
[DONE] Fanfounet/update access tokens

### PR DESCRIPTION
**Pull request by fanfounet**
Before sending your pull request, make sure the following are done :

* [x] You have read our [contribution guidelines](https://github.com/EsupPortail/Esup-Pod/blob/master/CONTRIBUTING.md).
* [x] Your PR targets the `develop` branch.
* [x] The title of your PR starts with `[WIP]` or `[DONE]`.
* [x] Add field token name
* [x] Add aside help
* [x] unit test

### CODE REVIEW

File: pod/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -8444,6 +8445,18 @@ msgstr "Catégorie"
...
+#| msgid "Meeting name"
+msgid "token name"
+msgstr "Nom de la session"
+
+#: pod/video/models.py
+#, fuzzy
+#| msgid "Change mode"
+msgid "change me"
+msgstr "Changer le mode d’édition"
+
```
Comment (AymericJak) [inline pod/locale/fr/LC_MESSAGES/django.po:8459]: À corriger ;)

+ enlever le fuzzy et #| ...

File: pod/video/models.py [hunk 2]
```diff
@@ -1835,6 +1835,7 @@ class VideoAccessToken(models.Model):
 
     video = models.ForeignKey(Video, on_delete=models.CASCADE)
     token = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(_("token name"), max_length=100, blank=True, default=_("change me"))
```
Comment (AymericJak) [inline pod/video/models.py:1838]: Peux tu spécifier comme suit : `verbose_name=_("Token name"), ...`

**Review by AymericJak — CHANGES_REQUESTED** [review]

**Review by AymericJak — APPROVED** [review]
RAS de mon côté.
Merci !

**Review by Badatos — APPROVED** [review]
une toute petite remarque, sinon Ok pour moi ;)

**Pull request merged**